forest/benchmarking/superoperator_conversion.py

```diff
@@ -132,6 +164,60 @@ def kraus2choi(kraus_ops: list):
         choi += np.kron(temp, temp.conj().T)
     return choi
 
+def chi2pauli_liouville(chi_matrix: np.ndarray):
+    r"""
+    Converts a chi matrix (aka a process matrix) to the Pauli Liouville representation.
+
+    :param chi: a D^2 x D^2 chi matrix
+    :return: dim**2 by dim**2 pauli-liouville matrix
+    """
+    dim = int(np.sqrt(chi_matrix.shape[0]))
+    n_qubits = int(np.log2(dim))
+    n_ops = dim**2
+
+    pl_matrix = np.zeros((dim**2, dim**2))
+
+    # loop over Pauli Liouville representaiton basis
+    for goodiesij in enumerate(zip(itertools.product(range(0, n_ops),range(0, n_ops)), itertools.product(n_qubit_pauli_basis(1),n_qubit_pauli_basis(1)))):
+        idx,jdx = goodiesij[1][0]
+        ((labi,Pi),(labj,Pj)) = goodiesij[1][1]
+        R_ij = 0
+        # sum over chi matrix elements
+        for goodieskl in enumerate(zip(itertools.product(range(0, n_ops),range(0, n_ops)), itertools.product(n_qubit_pauli_basis(1),n_qubit_pauli_basis(1)))):
+            kdx,ldx = goodieskl[1][0]
+            ((labk,Pk),(labl,Pl)) = goodieskl[1][1]
+            # (R_{\mathcal E})_{i,j} = \frac 1 d \sum_{k,l}\chi_{k,l} {\rm Tr}[ P_i P_k P_j P_l]
+            R_ij += chi_matrix[kdx,ldx]*np.trace(np.matmul(np.matmul(np.matmul(Pi, Pk), Pj), Pl))
+        pl_matrix[idx,jdx]=np.real(R_ij/dim)
+    return pl_matrix
```

Comment:
Similarly to my above comment, can we not more easily convert a chi matrix to a choi matrix using the change of basis matrices? That would replace this conversion which I find harder to parse. I might be missing something here though because I don't understand why the diagram 
pauli-liouville  <--->  liouville
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;     |       &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;     |
chi_matrix      <--->  choi matrix
doesn't totally 'commute'. E.g. I expected that you could go between p-l and chi by reshuffling, but that doesn't seem to be true, so maybe the basis change needs some shuffling in order to convert between chi and choi?